lib/PIClient/PrivacyIDEA.php

```diff
@@ -419,15 +544,14 @@ private function sendRequest(array $params, array $headers, string $httpMethod,
 	 */
 	public function log($level, $message): void
 	{
-		$context = ['app' => 'privacyIDEA'];
 		if ($level === 'debug') {
-			$this->logger->debug($message, $context);
+			logger('privacyIDEA')->info($message); // Temporarily use info level for debug messages
```

Comment:
how long is temporarily?

Reply:
[58729f8](https://github.com/privacyidea/privacyidea-nextcloud-app/pull/26/commits/58729f8480628752da6d54d4c6002ed29983b2e8)